CONFIGDOC.md

```diff
@@ -31,17 +31,17 @@ Once you install the plugin, **Use MATLAB version** appears in the **Build Envir
 
 In this example, the list includes two registered tools as well as the option for specifying a custom installation. If you select **Custom**, a **MATLAB root** box appears in the UI. You must enter the full path to your preferred MATLAB root folder in this box.
 
-![use_matlab_version_tool](https://user-images.githubusercontent.com/48831250/217575011-ab82e8bf-c673-4364-946a-cdd54eb75c39.png)
+![use_matlab_version_tool](https://github.com/mw-hrastega/Times-Table-App/assets/48831250/908748e6-afeb-40c1-a2f8-f2f6bde6f224)

```

Comment:
Images are linking to your personal repo. Are you planning to change this before merging?

Reply:
Good catch! Fixed.